cirq/google/sim/xmon_simulator_test.py

```diff
@@ -94,6 +94,35 @@ def simulate(simulator, circuit, scheduler, **kw):
     return simulator.simulate(program, **kw)
 
 
+def test_wavefunction():
+    simulator = cirq.google.XmonSimulator()
+
+    hadamard = cirq.Circuit.from_ops(cirq.H(Q1))
+    result = simulator.simulate(hadamard)
```

Comment:
Instead of using a gate, specify an initial state and simulate an empty circuit. This gives you complete control over what the output should be,

Reply:
You need a circuit which acts on a qubit in order for the initial state to make sense (your example as no qubits, but an initial state of a single qubit)

Reply:
I want to simulate an empty circuit per Craig's suggestion. I'm assuming that's a circuit that doesn't call any gates (e.g "a circuit which acts on a qubit").